inc/ThirdParty/Themes/Divi.php

```diff
@@ -108,6 +113,29 @@ public function add_divi_to_description( $disable_youtube_lazyload ) {
 		return $disable_youtube_lazyload;
 	}
 
+	/**
+	 * Disables setting explicit dimensions on images where Divi calculates height as percentage.
+	 *
+	 * @since 3.8.2
+	 *
+	 * @param array $images The array of images selected for adding image dimensions.
+	 *
+	 * @return array The array without images using data-height-percentage.
+	 */
+	public function disable_image_dimensions_height_percentage( array $images ) {
+		if ( ! self::is_divi() ) {
+			return $images;
+		}
+
+		foreach ( $images as $key => $image ) {
+			if ( false !== strpos( $image, 'data-height-percentage' ) ) {
+				unset( $images[ $key ] );
+			}
+		}
+
+		return array_values( $images );
```

Comment:
we don't care about returning array values only on the main filter usage as we won't use keys.
So u can adjust the test code only to use this `array_values` or not based on the fixture.